{"name": "Проверка принятия заказа без номера заказа", "status": "failed", "statusDetails": {"message": "AssertionError: Ожидаемый код ответа:400, фактический:404", "trace": "self = <test_accept_order.TestAcceptOrder object at 0x0000020625C0B7D0>, id_of_new_courier = 373844\n\n    @allure.title('Проверка принятия заказа без номера заказа')\n    @allure.description('При  принятии заказа без номера заказа код ответа: 400, сообщение в теле : \"Недостаточно '\n                        'данных для поиска')\n    def test_accept_order_without_order_id(self, id_of_new_courier):\n        accept_order = AcceptOrder()\n    \n        accept_order.accept_order(courier_id=id_of_new_courier)\n    \n>       accept_order.check_response_status_code_of_accept_order_without_courier_or_order_id()\n\ntests\\test_accept_order.py:44: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\nendpoints\\accept_order.py:48: in check_response_status_code_of_accept_order_without_courier_or_order_id\n    self.check_response_status_code(self.STATUS_CODE_OF_ACCEPT_ORDER_WITHOUT_ORDER_OR_COURIER_ID)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <endpoints.accept_order.AcceptOrder object at 0x0000020625D14F20>, status_code = 400\n\n    def check_response_status_code(self, status_code):\n>       assert self.response.status_code == status_code, (f'Ожидаемый код ответа:{status_code}, '\n                                                          f'фактический:{self.response.status_code}')\nE       AssertionError: Ожидаемый код ответа:400, фактический:404\n\nendpoints\\base_point.py:12: AssertionError"}, "description": "При  принятии заказа без номера заказа код ответа: 400, сообщение в теле : \"Недостаточно данных для поиска", "steps": [{"name": "Принять заказ", "status": "passed", "parameters": [{"name": "order_id", "value": "None"}, {"name": "courier_id", "value": "373844"}], "start": 1724956345020, "stop": 1724956345146}, {"name": "Проверка статус кода принятия заказа без id курьера", "status": "failed", "statusDetails": {"message": "AssertionError: Ожидаемый код ответа:400, фактический:404\n", "trace": "  File \"C:\\Users\\MSI\\PycharmProjects\\Sprint_7\\venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"C:\\Users\\MSI\\PycharmProjects\\Sprint_7\\endpoints\\accept_order.py\", line 48, in check_response_status_code_of_accept_order_without_courier_or_order_id\n    self.check_response_status_code(self.STATUS_CODE_OF_ACCEPT_ORDER_WITHOUT_ORDER_OR_COURIER_ID)\n  File \"C:\\Users\\MSI\\PycharmProjects\\Sprint_7\\endpoints\\base_point.py\", line 12, in check_response_status_code\n    assert self.response.status_code == status_code, (f'Ожидаемый код ответа:{status_code}, '\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n"}, "start": 1724956345146, "stop": 1724956345147}], "start": 1724956345019, "stop": 1724956345152, "uuid": "8b4e0e10-894f-43e0-8c64-dc56ac6850dc", "historyId": "af66eded73cf39e7112a84e826dffdfa", "testCaseId": "af66eded73cf39e7112a84e826dffdfa", "fullName": "tests.test_accept_order.TestAcceptOrder#test_accept_order_without_order_id", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_accept_order"}, {"name": "subSuite", "value": "TestAcceptOrder"}, {"name": "host", "value": "DESKTOP-AS1GOJN"}, {"name": "thread", "value": "5404-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_accept_order"}]}